repos/system_upgrade/common/actors/checkinsightsautoregister/libraries/checkinsightsautoregister.py

```diff
@@ -24,9 +24,9 @@ def _ensure_package(package):
 def _report_registration_info(installing_client):
     pkg_msg = " The '{}' package required for the registration will be installed during the upgrade."
 
-    title = "Automatic registration into Red Hat Insights"
+    title = "Automatic registration into Red Hat Lightspeed"
```

Comment:
@mhuth I just realized the modified title of the report which leads to a different key of the report. I suggest to add few lines below following line to ensure the report key stays as it is:
```
        reporting.Key('693963253195f418526f045b6d630a1f4c7a193d')
```

Reply:
Done!